packages/next/client/index.js

```diff
@@ -407,9 +410,19 @@ function AppContainer({ children }) {
       }
     >
       <RouterContext.Provider value={makePublicRouterInstance(router)}>
-        <HeadManagerContext.Provider value={headManager.updateHead}>
-          {children}
-        </HeadManagerContext.Provider>
+        <FallbackContext.Provider
+          value={
+            // Fallback is true iff server-rendered page was a fallback
```

Comment:
```suggestion
            // Fallback is true if server-rendered page was a fallback
```

Reply:
This [iffs](https://www.merriam-webster.com/dictionary/iff) is intentional, i.e. "if and only if".